Kickstarter-iOS/Features/ConfirmDetails/Controllers/ConfirmDetailsViewController.swift

```diff
@@ -301,6 +318,18 @@ final class ConfirmDetailsViewController: UIViewController, MessageBannerViewCon
 
   // MARK: - Functions
 
+  private func goToLoginSignup(with intent: LoginIntent, project: Project, reward: Reward?) {
+    let loginSignupViewController = LoginToutViewController.configuredWith(
+      loginIntent: intent,
+      project: project,
+      reward: reward
+    )
```

Comment:
At first glance, it looks like LoginToutViewController doesn't actually use the project or reward being passed in, but I found that they're specifically for analytics.

Reply:
I've noticed this in quite a few places - a a view model requiring way more information than seems initially necessary, because it gets passed through to analytics. This is something we might consider for future refactoring. The way we pass around "contexts" in some places feels like a better model for this.